### PR TITLE
[ONNXIFI] Fix resnet50 integration test

### PR DESCRIPTION
**Pull request by yinghai**
*Description*:
The assumption that ONNX model only contains one node doesn't hold because Caffe2 op might be exported into multiple ONNX nodes. 
*Testing*:
Run the ONNXIFI integration test from Pytorch repo:
pytest python/onnx/test_onnxifi.py
And it should pass. 

*Documentation*:
[Optional Fixes #issue]

Please see a detailed explanation of how to fill out the fields in the relevant sections in PULL_REQUEST.md.

### CODE REVIEW

**Comment by rdzhabarov** [comment]
Please, run clang-format on the changes and I'll merge after that.

**Comment by yinghai** [comment]
What's wrong with my clang-format? It fails the clang format check....

File: lib/Importer/ONNXIFIModelLoader.cpp [hunk 1]
```diff
@@ -146,55 +146,51 @@ ONNXIFIModelLoader::parseOperator(const void *onnxModel, size_t onnxModelSize) {
 
   ONNX_NAMESPACE::GraphProto graph = modelDef.graph();
 
-  // Only single operator is allowed to be in the onnxModel.
-  if (graph.node_size() != 1) {
-    return result;
-  }
-
-  const std::string &operation = graph.node(0).op_type();
```
Comment (rdzhabarov) [inline lib/Importer/ONNXIFIModelLoader.cpp]: two empty lines here

Comment (rdzhabarov) [reply lib/Importer/ONNXIFIModelLoader.cpp]: you can run: ```./utils/format.sh fix```

**Comment by yinghai** [comment]
> Why did it fail in the first place? I assume returning "non supported" operator should be properly handled in the graph-cutting.

I believe these lines were added later...

**Comment by rdzhabarov** [comment]
> I believe these lines were added later...

I meant to say that if ONNXIFI backend reports that the operator (in this case several operators) is not supported it should not cause the failure we are seeing now in the integration test.
I worry if there is a bigger problem which is hidden now by returning "operator is supported".

**Comment by yinghai** [comment]
> I meant to say that if ONNXIFI backend reports that the operator (in this case several operators) is not supported it should not cause the failure we are seeing now in the integration test.

This is because the issue of the test itself. It uses input(0) as tensor to feed but in this case it's not the tensor we want hence the failure.

**Pull request merged**